### ISSUE TITLE
Make winston plugin

### ISSUE THREAD

**Issue by financedemo** [kilianc/rtail#59]
I think first is to make it work programmatically

**Comment by kilianc** [comment]
Yeah regardless I need to make it work programmatically, it will help writing tests. The winston plugin then could just require the lib.
